bin/ch-grow.py.in

```diff
@@ -465,6 +467,15 @@ class CLI_Print_Storage(ch.CLI_Action_Exit):
       print(namespace.storage)
       sys.exit(0)
 
+class CLI_List_Images(ch.CLI_Action_Exit):
+
+   def __call__(self, _, namespace, *args, **kwargs):
+      print("IMAGE") 
+      p = pathlib.Path(namespace.storage + '/img')
+      for child in p.iterdir():
+          print(child.name)
```

Comment:
Probably should translate this back to the canonical form, e.g., substitute `%` back to `/`. The easy way to do that might be to parse an `Image_Ref`. If that fails, you probably found a bogus directory.